lib/turn.rb

```diff
@@ -0,0 +1,71 @@
+class Turn
+    attr_reader :player1, :player2, :spoils_of_war, :winner
+    def initialize(player1, player2)
+      @player1 = player1
+      @player2 = player2
+      @spoils_of_war = []
+      @winner = winner
```

Comment:
We won't want to instantiate with a `winner` because we calculate that after the turn is complete. The `winner` method on line 20 is sufficient to give us what we need.